superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -219,6 +240,15 @@ export default function DataSourcePanel({
                 )}
               </LabelContainer>
             ))}
+            {lists.metrics.length > DEFAULT_MAX_METRIC_LENGTH ? (
+              <ButtonContainer>
+                <Button onClick={() => setShowAllMetrics(!showAllMetrics)}>
+                  {showAllMetrics ? t('Show Less...') : t('Show all...')}
```

Comment:
One more giant nitpick. 
```suggestion
                  {showAllMetrics ? t('Show less...') : t('Show all...')}
```